src/object.rs

```diff
@@ -754,6 +755,10 @@ impl Metadata {
         self
     }
 
+    pub(crate) fn name(&self) -> Option<&str> {
```

Comment:
- Remove the `(crate)` so that we can use it as a public API.
- And public API requires doc